src/btest.toit

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2021 Ekorau LLC
```

Comment:
It's always nice to have an "entry" library.
If the package is called `btest` then users can just write `import btest` and reach this file.